addons/sys_intercom/fnc_setRxTxCapabilities.sqf

```diff
@@ -18,11 +18,14 @@
  */
 #include "script_component.hpp"
 
-params ["_rackId", "_vehicle", "_unit", "_functionality"];
+params ["_radioId", "_vehicle", "_unit", "_functionality", ["_rackId", objNull]];
 
 private _found = false;
 private _vehicleConfiguration = _vehicle getVariable [QGVAR(rackTxRxConfig), []];
-_rackId = toLower _rackId;
+
+if (isNull _rackId) then {
+    private _rackId = toLower ([toLower _radioId] call EFUNC(sys_rack,getRackFromRadio));
```

Comment:
Yes :) indeed. This code is still WIP however and it lacks revision from my side.